index.html

```diff
@@ -262,7 +262,7 @@
                 <li><a id="restoreIcon" class="tooltipped" data-position="bottom"><i class="material-icons md-48">restore_from_trash</i></a></li>
                 <li><a id="beginnerMode" class="tooltipped" data-position="bottom"><i id="begIconText" class="material-icons md-48" >star</i></a></li>
                 <li><a id="advancedMode" class="tooltipped" data-position="bottom"><i id="advIconText" class="material-icons md-48" >star_border</i></a></li>
-                <li><a id="languageSelectIcon" class="tooltipped" data-position="bottom"  data-tooltip=""><i class="material-icons md-48">translate</i></a></li>
+                <li><a id="languageSelectIcon" class="tooltipped dropdown-trigger" data-position="bottom" data-delay="100" data-activates='languagedropdown><i class="material-icons md-48">translate</i></a></li>
```

Comment:
- You missed a quote after languagedropdown
- Could you remove the data-delay attribute? Currently the data-delay is set centrally in toolbar.js and it may cause some confusion further down the line.
https://github.com/sugarlabs/musicblocks/blob/2afb96ff345dad377742239b6d7a5825300129e3/js/toolbar.js#L344